src/enums/ld-flags.ts

```diff
@@ -19,5 +19,4 @@ export enum LDFlags {
   SupportedDissolutionEntities = 'supported-dissolution-entities',
   SupportedRestorationEntities = 'supported-restoration-entities',
   SupportedPutBackOnEntities = 'supported-put-back-on-entities',
-  UseBusinessApiGwURL = 'use-business-api-gw-url',
```

Comment:
Obsolete. This is currently True in all environments (except Sandbox, where I believe this UI is not used).